tasks/update-cnb-dependency/dependencies.go

```diff
@@ -16,7 +16,7 @@ type Dependency struct {
 	Name         string   `toml:"name,omitempty"`
 	SHA256       string   `toml:"sha256"`
 	Source       string   `toml:"source,omitempty"`
-	SourceSHA256 string   `toml:"source_sha256,omitempty"`
+	SourceSHA256 string   `mapstructure:"source_sha256" toml:"source_sha256,omitempty"`
```

Comment:
Why don't any other of the keys have a mapstructure link? We only need it if it's a non-intuitive match?

Reply:
Right. When decoding json, yaml, or mapstructure (but not toml), the key name of the struct has to be a case-insensitive match of the data you're decoding. Since there's an underscore in this case, you have to tell it explicitly what the kay name is.

https://play.golang.org/p/g7Bq-vL2L4n